CHANGELOG.md

```diff
@@ -91,6 +91,12 @@ v0.31.0 (2023-01-31)
 
 - Introduce global configuration for logs. (@jcreixell)
 
+- Integrations: Introduce `cloudwatch metrics` integration. (@thepalbi)
```

Comment:
I think this needs to go somewhere else

Reply:
yess! My bad, it seems it moved during a rebase or something